plugins/modules/dcnm_service_policy.py

```diff
@@ -0,0 +1,1962 @@
+      they will be updated if possible.
+    - Properties that can be managed by the module but are not specified
+      in the playbook will be deleted or defaulted if possible.
+    - Service Peerings that are not specified in the playbook will be deleted.
+
+Deleted:
+  Service Peerings defined in the playbook will be deleted.
+
+Query:
+  Returns the current DCNM state for the Service Peerings listed in the playbook.
+
+CREATING SERVICE POLICIES
+=========================
+
+- name: Create service policy including all optional objects
+  cisco.dcnm.dcnm_service_policy:
+    fabric: test_fabric
+    attached_fabric: external
+    attach: true
+    deploy: true
+    state: merged
+    config:
+      - name: service_policy_1
+        node_name: it-sn-1
+        rp_name: it-fw-rp1
+        src_vrf: vrf_11
+        dest_vrf: vrf_11
+        src_network: net_11
+        dest_network: net_12
+        next_hop: 192.161.1.100
+        rev_next_hop: 192.161.2.100
+        reverse: true
+        policy:
+          proto: tcp
+          src_port: any
+          dest_port: 22
+          action: permit
+          next_hop_option:  none
+          acl_name: fwd_acl_10
+          rev_acl_name: rev_acl_10
+          route_map_num: 101
+          rev_route_map_num: 102
+
+DELETE SERVICE POLICIES
+=======================
+
+1. With Policy Name and Node name
+
+Deletes the specific service policy specified from the given node
+
+- name: Delete service policies with policy name and node name
+  cisco.dcnm.dcnm_service_policy:
+    fabric: test_fabric
+    attached_fabric: external
+    state: deleted
+    config:
+      - name: service_policy_1
+        node_name: it-sn-1
+            
+      - name: service_policy_2
+        node_name: it-sn-2
+            
+      - name: service_policy_3
+        node_name: it-sn-2
+            
+      - name: service_policy_4
+        node_name: it-sn-2
+              
+      - name: service_policy_5
+        node_name: it-sn-2
+
+2. With Node name alone
+
+Deletes all service policies from the specified nodes
+
+- name: Delete service policies with Node names
+  cisco.dcnm.dcnm_service_policy:
+    fabric: test_fabric
+    attached_fabric: external
+    state: deleted
+    config:
+      - node_name: it-sn-1
+      - node_name: it-sn-2
+
+3. With Node name and RP name
+
+   Deletes all service policies under the specified route peering and node
+
+- name: Delete service policies with Node name and RP name
+  cisco.dcnm.dcnm_service_policy:
+    fabric: test_fabric
+    attached_fabric: external
+    state: deleted
+    config:
+      - node_name: it-sn-1
+        rp_name: it-fw-rp1
+
+      - node_name: it-sn-2
+        rp_name: it-fw-rp2
+
+4. Without config
+
+   Deletes all service policies on the given fabric and attached fabirc
+
+- name: Delete service policies without config
+  cisco.dcnm.dcnm_service_policy:
+    fabric: test_fabirc
```

Comment:
`/fabirc/fabric/`. Check for other places where this typo might exist.

Reply:
Done